libraries/Microsoft.Bot.Builder.AI.QnA/Models/QnAMakerTraceInfo.cs

```diff
@@ -119,5 +119,30 @@ public class QnAMakerTraceInfo
 #pragma warning disable CA1819 // Properties should not return arrays (this property is obsolete and we won't change it)
         public Metadata[] MetadataBoost { get; set; }
 #pragma warning restore CA1819 // Properties should not return arrays
+
+        /// <summary>
+        /// Gets or sets the list of sources used to filter QnA Maker to filter results.
+        /// </summary>
+        /// <value>
+        /// An array of string.
+        /// </value>
+        [JsonProperty("filters")]
+        public Models.Filters Filters { get; set; }
+
+        /// <summary>
+        /// Gets or sets AnswerSpanRequest.
+        /// </summary>
+        /// <value>
+        /// The AnswerSpanRequest.
+        /// </value>
+        [JsonProperty("answerSpanRequest")]
+        public AnswerSpanRequest AnswerSpanRequest { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to enable Query over Unstructured Sources (Optional).
```

Comment:
This is a bool property and can't be "optional" as the documentation says. If it is not set, it will default to false.